Dashboard/Views/AboutView.axaml

```diff
@@ -0,0 +1,8 @@
+﻿<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:Class="Dashboard.Views.AboutView">
+    Welcome to Avalonia!
```

Comment:
One thing I have noticed is these controls seem to scale to fit the page 1:1, meaning the text gets quite big. Is there a way for us to handle this?

Reply:
![image](https://github.com/user-attachments/assets/02505450-c41b-464a-b3bf-8f5bc157a2bf)

vs main branch

![image](https://github.com/user-attachments/assets/b5ef6cca-24f8-44bc-9aa9-73cd56f087b5)

Reply:
I currently have the content in a ViewBox which scales the content to be 'Uniform' manner but I can leave the scaling to be done purely by the Views themselves. Only thing with this is, I'm not sure how they handle scaling within ContentControls.

Reply:
I'm not too picky how it gets done, but to confirm we can fix the scaling so it looks somewhat regular?

Reply:
I have removed the ViewBox, I tested around with the views doing the scaling themselves and it looks like it works like normal.
https://github.com/nickowen44/AV-Datalogger/pull/6/commits/d612a8dfdb08970adabc00d5a4e1fb42ce5e90d0

Reply:
yeah that looks good, ty :)